www/front_src/src/Resources/Details/tabs/Details/DetailsCard/index.tsx

```diff
@@ -1,27 +1,52 @@
 import * as React from 'react';
 
-import { Card, CardContent, Typography, Grid } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+
+import { Typography, makeStyles, Tooltip } from '@material-ui/core';
+import IconCheck from '@material-ui/icons/Check';
+
+import { labelActive } from '../../../../translatedLabels';
+import Card from '../Card';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    height: '100%',
+  },
+  content: {
```

Comment:
This one is not used in this component

Reply:
We might want to check out something like: https://github.com/jens-ox/eslint-plugin-material-ui-unused-classes